src/_data/techsponsors.json

```diff
@@ -17,4 +17,4 @@
         "image": "/assets/images/techsponsors/1password-icon.svg",
         "donation": "password management"
     }
-]
```

Comment:
these files were added to prettier ignore right 🤔

Reply:
this is weird issue while reverting those json files, u can see in the changes it has 

- ]
and
+ ]

so they should cancel each other out.

Reply:
They're missing a newline character at the end.